src/app_neo4j_queries.py

```diff
@@ -670,6 +670,44 @@ def get_associated_organs_from_dataset(neo4j_driver, dataset_uuid):
 
     return results
 
+def get_associated_samples_from_dataset(neo4j_driver, dataset_uuid):
```

Comment:
Please add the comment block above the function declaration

Reply:
OK!